Package.swift

```diff
@@ -18,7 +18,7 @@ let package = Package(
         .package(name: "Ink", url: "https://github.com/johnsundell/ink.git", from: "0.2.0"),
         .package(name: "Plot", url: "https://github.com/johnsundell/plot.git", from: "0.4.0"),
         .package(name: "Files", url: "https://github.com/johnsundell/files.git", from: "4.0.0"),
-        .package(name: "Codextended", url: "https://github.com/johnsundell/codextended.git", from: "0.1.0"),
+        .package(name: "Codextended", url: "https://github.com/addisonwebb/codextended.git", .branch("update-any-date-formatter")),
```

Comment:
This will be updated once the Codextended PR is merged. https://github.com/JohnSundell/Codextended/pull/15